projects/bring/index.js

```diff
@@ -101,12 +102,12 @@ module.exports = {
   },
   bsc: {
     staking: stakings(
-      [chefContractsNew_bsc, chefContractsOld_bsc],
+      [chefContractsNew_bsc, chefContractsOld_bsc, chefContractsNew_bsc1],
       BRNG_bsc,
       "bsc"
     ),
     pool2: pool2s(
-      [chefContractsNew_bsc, chefContractsOld_bsc],
+      [chefContractsNew_bsc, chefContractsOld_bsc, chefContractsNew_bsc1],
```

Comment:
Not sure if I should add the new chefContractsNew_bsc1 on this line(110)
But it contains some Pancakeswap LP

Reply:
yeah its good